src/legacy/server/saved_objects/saved_objects_mixin.test.js

```diff
@@ -0,0 +1,248 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { savedObjectsMixin } from './saved_objects_mixin';
+import sinon from 'sinon';
+
+describe('Saved Objects Mixin', () => {
+  let mockKbnServer;
+  let mockServer;
+  const mockCallCluster = sinon.spy();
```

Comment:
I agree with @mikecote, please use the built in [jest mocks](https://jestjs.io/docs/en/mock-function-api) rather than sinon in jest tests.

Reply:
I will get these converted over to jest mocks in a follow up PR.